cmd/keycloak-debug/dumpgroups.go

```diff
@@ -18,6 +18,7 @@ type DumpGroupsCmd struct {
 	KeycloakClientID     string `kong:"default='service-api',env='KEYCLOAK_SERVICE_API_CLIENT_ID',help='Keycloak OAuth2 Client ID'"`
 	KeycloakClientSecret string `kong:"required,env='KEYCLOAK_SERVICE_API_CLIENT_SECRET',help='Keycloak OAuth2 Client Secret'"`
 	KeycloakRateLimit    int    `kong:"default=10,env='KEYCLOAK_RATE_LIMIT',help='Keycloak API Rate Limit (requests/second)'"`
+	KeycloakPageSize     int    `kong:"default=400,env='KEYCLOAK_PAGE_SIZE',help='Keycloak API Page Size'"`
```

Comment:
In the "happy path" case for 1k results this is going to be 3x the number of requests, and increase latency.

Would it make sense to keep the 1k default value and reduce it to 400 operationally until the slow Keycloak requests are resolved?